packages/wrangler/src/index.tsx

```diff
@@ -68,9 +75,25 @@ async function readConfig(configPath?: string): Promise<Config> {
     );
   }
 
+  if ("unsafe" in config) {
+    console.warn(
+      "'unsafe' fields are experimental and may change or break at any time."
+    );
+  }
+
   // todo: validate, add defaults
   // let's just do some basics for now
 
+  for (const binding of config.unsafe?.bindings ?? []) {
+    if (knownBindings.includes(binding.type)) {
+      console.warn(
+        `Raw '${binding.type}' bindings are not directly supported by wrangler. Consider migrating to a ` +
```

Comment:
```suggestion
        `Unsafe '${binding.type}' bindings are not directly supported by wrangler. Consider migrating to a ` +
```

Reply:
I was thinking for documentation purposes we'd call these "raw bindings" (because that's what it is) and going forward any experimental features would be placed in the `unsafe` table for consistency. I'd be open to writing the warning as `Unsafe raw 'x' bindings` though. Just thinking if we take this path that "unsafe bindings" are vague because other binding types could be added to the unsafe table but wouldn't be raw bindings.

Reply:
They're unsafe because we may break this at any point and don't commit to supporting it forever, so the word unsafe should probably be there. If we want to use this for actually stable bindings, then we should add them as first class bindings, regardless of whether they're internal only. We can simply not document them.